components/patient-data/ui/src/main/resources/PhenoTips/UIX_Field__gene-variants.xml

```diff
@@ -137,16 +138,19 @@
         var infoTool = new Element('span', {'class' : 'fa fa-info-circle xHelpButton gene-info', 'title' : geneNames[i]});
         var icon_helpController = new PhenoTips.widgets.HelpButton(infoTool);
         var triangle = new Element('span', {'class': "fa fa-exclamation-triangle", title: '$services.localization.render('phenotips.patientSheetCode.termSuggest.nonStandardGenesymbol')'}).update(' ');
-        if (isSuggestSelected)
+        if (isSuggestSelected) {
+          element.removeClassName(nonstandard);
           element.insert({after: infoTool});
-        else
+        } else {
+          element.addClassName(nonstandard);
           element.up().insert({top: triangle});
```

Comment:
I'm fine not to show a warning sign, but imo we should not show an info button either @buske